ui/src/pages/server-types/server-types.tsx

```diff
@@ -79,11 +79,12 @@ export const ServerTypes = (): React.ReactElement => {
   };
   const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
     event.preventDefault();
+    const displayName = currentFormInput?.display_name || '';
     const payload = {
-      servername: currentServerName || '',
+      servername: currentServerName || displayName,
       user_options: {
         jhub_app: true,
-        name: currentServerName || '',
+        name: currentServerName || displayName,
```

Comment:
This field is not required and doesn't exists in the `UserOptions` model: https://github.com/nebari-dev/jhub-apps/blob/8d424ae91d8ee31de61d752f30f22e8249d30ce1/jhub_apps/service/models.py#L52

Reply:
Will add something to the backlog to clean this up.